### PR TITLE
feat(docs): add run agents on agentverse example

### PR DESCRIPTION
**Pull request by FelixNicolaeBucsa**

### CODE REVIEW

**Review by devjsc — DISMISSED** [review]

File: pages/examples/intermediate/running-an-agent-on-agentverse.mdx [hunk 1]
```diff
@@ -0,0 +1,178 @@
...
+
+## Create Agents hosted on the Agentverse
+
+It is possible for you to create and host any type of agent you want using the [My Agents ↗️](/concepts/agent-services/agent-hosting) tab. Managed agents are currently a beta feature and therefore do not fully support the entire [uagents ↗️](/concepts/agents/agents) library toolset for development. Check out the [Agentverse: allowed imports ↗️](/guides/agentverse/allowed-imports) guide for a better understanding of the packages available for development of Agents on the Agentverse.
+
+You can choose between creating agents from a **Blank Agent** script or using **Skeleton Agent** code as an initial template.
+
+### Create and run an agent on Agentverse
+
+First of all, navigate towards the [Agentverse ↗️](https://agentverse.ai/), sign in and choose the **My Agents** tab. Here, click on the **+ New Agent** button and choose **Blank Agent**. Provide the needed details including the agent's name.
```
Comment (devjsc) [inline pages/examples/intermediate/running-an-agent-on-agentverse.mdx]: What is the difference between this example and a guide?

**Review by devjsc — CHANGES_REQUESTED** [review]

File: pages/examples/intermediate/running-an-agent-on-agentverse.mdx (hunk 1 above)
Comment (FelixNicolaeBucsa) [reply pages/examples/intermediate/running-an-agent-on-agentverse.mdx]: I thought this example had to gather ways to create and run agents on Agentverse. I can remove some text around the code in these paragraphs to make it look sharper but let me know what you were thinking for content to be in it

File: pages/examples/intermediate/running-an-agent-on-agentverse.mdx [hunk 2]
```diff
@@ -0,0 +1,178 @@
...
+
+    ```py copy filename="agent.py"
+    from uagents import Agent, Context, Model, Protocol
+
+    class Message(Model):
+        message: str
+
+    SEED_PHRASE = "put_your_seed_phrase_here"
+
+    print(f"Your agent's address is: {agent.address}")
```
Comment (gautamgambhir97) [inline pages/examples/intermediate/running-an-agent-on-agentverse.mdx]: @FelixNicolaeBucsa this should be come after the agent initialisation else the code will through error

**Review by gautamgambhir97 — CHANGES_REQUESTED** [review]

File: pages/examples/intermediate/running-an-agent-on-agentverse.mdx (hunk 2 above)
Comment (FelixNicolaeBucsa) [reply pages/examples/intermediate/running-an-agent-on-agentverse.mdx]: Am fixing it rn

Comment (FelixNicolaeBucsa) [reply pages/examples/intermediate/running-an-agent-on-agentverse.mdx]: Solved

**Pull request merged**